### PR TITLE
Create journal-week-4.md and tutorial-week-4.md

### PR DESCRIPTION
**Pull request by ShayHall**

### CODE REVIEW

**Comment by rosalsm** [comment]
Lots of try and fail, this is the point! :+1:

**Comment by s4dmw** [comment]
journal - i really like the underlying message that failure is a part of learning this stuff!
tutorial - interesting idea/topic...i look forward to seeing you expand on this
:+1:

**Comment by jeremyrist** [comment]
Takin' em back on the tutorial! :+1: Good journal too haha

File: journal-week-4.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+I tap my foot.
+
+I lift my hand to refresh the page.
+
+And I think...
+> Will it work?  Did I do it?  Have I solved the puzzle?  Am I done?  Can I go to sleep?
+
+These are just a few of the many questions that run explicitly through my mind and bubble up silently through my being in the form of an anxiety that has no concept of real hope.  Like a web page after I've CSSed it to pieces, my emotions are all over the place.
+
+I can sit for hours, trying things that seem reasonable, attempting various combinations of `.` `;` `()` `{}` `[]` that begin to suspiciously look more and more like cartoon cussing. `&$!%`
```
Comment (kellymurray) [inline journal-week-4.md]: $^%($%($&!!!!!

**Comment by jmcreasman** [comment]
Loved the journal entry, very much relate to it

**Pull request merged**